addons/udes_security/controllers/authSignupHome.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from odoo.http import request
+from odoo import http, exceptions
+import odoo.addons.web.controllers.main as main
+import odoo.addons.auth_signup.controllers.main as authmain
+from odoo.addons.web.controllers.main import ensure_db
+from ..common import remove_external_redirect
+
+
+class AuthSignupHome(authmain.AuthSignupHome):
+
+    # Override web_login function
```

Comment:
Again, change to a docstring. I would also like to see a one-sentence explanation of what this doing to the web_login function. e.g. Override web_login function to remove open redirect.